src/mpyl/steps/deploy/k8s/chart.py

```diff
@@ -205,14 +214,49 @@ def to_spark_config_map(self) -> V1ConfigMap:
             metadata=self._to_object_meta()
         )
 
+    def __find_default_port(self) -> int:
+        found = next(iter(self.mappings.keys()))
+        if found:
+            return int(found)
+        raise KeyError("No default port found. Did you define a port mapping?")
+
+    def create_host_wrappers(self) -> list[HostWrapper]:
+        default_hosts: list[Host] = Traefik.from_config(self.config_defaults.treafik_defaults).hosts
+
+        hosts: list[Host] = self.deployment.traefik.hosts if self.deployment.traefik else []
+
+        first_host = next(iter(hosts), None)
+        service_port = first_host.service_port if first_host and first_host.service_port else self.__find_default_port()
+
+        def to_white_list(configured: Optional[TargetProperty[list[str]]]) -> list[str]:
+            white_lists = configured.get_value(self.target) if configured else self.config_defaults.white_lists[
+                'default']
```

Comment:
Just wondering: is it possible with pylint to also define formatting rules like with eslint/prettier? I for one would find it easier while coding if it would tell me where we like our line breaks, how many tabs/spaces and how to align stuff. And it would make the code look much more consistent/clean overall.

Reply:
Yeah, would be nice. Did you find anything out wrt this?

Reply:
https://black.readthedocs.io/en/latest/ maybe?

Reply:
https://vandebron.atlassian.net/browse/TECH-400

Reply:
Worth trying out 👍